packages/jellyfish-api-core/__tests__/category/loan/paybackLoan.test.ts

```diff
@@ -752,14 +836,279 @@ describe('paybackLoan failed', () => {
     await expect(promise).rejects.toThrow(RpcApiError)
     await expect(promise).rejects.toThrow('tx must have at least one input from token owner')
   })
+})
 
-  it('should fail paybackLoan if resulted in zero interest loan', async () => {
-    const promise = bob.rpc.loan.paybackLoan({
-      vaultId: bobVaultId,
-      amounts: '40@TSLA',
-      from: bobloanAddr
+describe('paybackLoan before FortCanningHeight', () => {
+  let tGroupFCH: TestingGroup
+  let loanTokenMinter: Testing
+  let testing: Testing
+  let colAddr: string
+  let vaultId: string
+  const tslaLoanAmount = 10
+
+  async function setupPreFCHContainer (): Promise<void> {
+    tGroupFCH = TestingGroup.create(2)
+    const hardForks: HardForkOptions[] = [{ name: 'fortcanninghillheight', blockHeight: 1000000 }]
+    await tGroupFCH.start({ hardForks })
```

Comment:
not sure if we want to introduce this only for hard fork options.
We already have 2 other methods to make this happen.
1. creating a custom container like here -> https://github.com/DeFiCh/jellyfish/blob/main/packages/jellyfish-api-core/__tests__/category/governance/governance_container.ts
2. set the config and restart (not exactly suitable for this test. but can be done.)

I'm ok with refactoring `StartOptions` to accept any kind of flags with more generalization. so that we can use it for more test cases in the future.
any comments @fuxingloh ?

Reply:
ain flags -> https://github.com/DeFiCh/ain/blob/b63a433a5d1f50f385748aa2524397e9de91bde6/src/init.cpp#L361

Reply:
> I'm ok with refactoring `StartOptions` to accept any kind of flags with more generalization. so that we can use it for more test cases in the future. any comments @fuxingloh ?

Yes, I prefer this (generalization) over extending HardForkOptions as another feature of Flag.